vllm/model_executor/layers/vocab_parallel_embedding.py

```diff
@@ -96,8 +122,12 @@ def forward(self, input_):
             masked_input[input_mask] = 0
         else:
             masked_input = input_
-            # Get the embeddings.
+
+        # Get the embeddings.
+        # TODO: linear_method base class does not have embedding api
+        assert not self.linear_method.QUANTIZED
         output_parallel = F.embedding(masked_input, self.weight)
```

Comment:
@robertgshaw2-neuralmagic  We are running into this issue right now. `linear_method` base has no `embedding` api so we can't just call `linear_method.embedding(masked_input)`. This can be adde to `UnquantizedLinearMethod` but we don't have the tool/skills to add this api to gptq/other qunatized methods. 

Should we leave this piece of code as-is with TODO comment?

Reply:
We can leave as a TODO and address in another PR

Do you have any example models with:
- quantized lm head, no quantized embedding
- quantized lm head, quantized embedding